BrainPortal/app/models/bourreau.rb

```diff
@@ -393,6 +402,18 @@ def send_command_alter_tasks(tasks,new_task_status,extra_remote_commands = {})
 
   protected
 
+  # This comaand is a bit dangerous: it will first trigger a +stop_workers+
+  # command to be sent locally, and then the rails app will kill itself with
+  # a TERM signal, thus exiting right after the current request. The user should
+  # really make sure the workers are inactive and there is no other background
+  # activity.
```

Comment:
Is this to do in the future, or is there a reason you are not making this check?

Reply:
Because we can't really make the check: once background activity is started it's forked into a separate process completely disconnected from the rails server (using double fork). So the rails server doesn't know if a background process is also using the DB.